Source/JavaScriptCore/runtime/UGPRPair.h

```diff
@@ -38,7 +38,7 @@ struct UGPRPair {
     UCPURegister first;
     UCPURegister second;
 };
-static_assert(sizeof(UGPRPair) >= sizeof(void*) * 2, "UGPRPair should fit in two machine registers");
+static_assert(sizeof(UGPRPair) == sizeof(void*) * 2, "UGPRPair should fit in two machine registers");
```

Comment:
This doesn't work on the Apple Watch build, which is surprising to me.

Reply:
This does not work on Apple Watch since it is using ARM64_32 ABI. You can change `void*` to UCPURegister.